examples/Fusion.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE TemplateHaskell, CPP #-}
+{-# OPTIONS_GHC -dsuppress-all -funfolding-use-threshold=100 #-}
```

Comment:
the usual cheat code with `inspection-testing`

Reply:
I suspect it could do the trick for generics-lens case too.

Reply:
I first tried with 100, that broke GHC-8.2. With 120 every GHC seems to be happy. I think this just proves the point of necessity of a tool like `inspection-testing` :)